From: Evgeni Golov <sargentd@die-welt.net>
Date: 2008-12-15 20:14:24 +0100
Subject: Hurd does not have PATH_MAX
License: GPL-2+

There is no PATH_MAX on Hurd (see [1]), thus define it to something sane.

[1] http://www.debian.org/ports/hurd/hurd-devel-debian

Index: kanatest-0.4.6/src/gui_utils.h
===================================================================
--- kanatest-0.4.6.orig/src/gui_utils.h	2008-12-15 19:13:45.000000000 +0000
+++ kanatest-0.4.6/src/gui_utils.h	2008-12-15 19:14:24.000000000 +0000
@@ -37,5 +37,9 @@
 void        gui_url_insert_link                 (GSList *links_list, gint *link_index, GtkWidget *textview, GtkTextIter *iter, gchar *color, gint font_size, gchar *text, gboolean center, GUI *appGUI);
 void        gui_url_remove_links                (GSList *links_list, gint *link_index);
 
+#ifndef PATH_MAX
+ #define PATH_MAX 4096
+#endif
+
 #endif /* _GUI_UTILS_H */
 
Index: kanatest-0.4.6/src/prefs.h
===================================================================
--- kanatest-0.4.6.orig/src/prefs.h	2008-12-15 19:19:53.000000000 +0000
+++ kanatest-0.4.6/src/prefs.h	2008-12-15 19:20:30.000000000 +0000
@@ -34,6 +34,10 @@
 #define MAX_COLORNAME_LEN   64
 #define MAX_BUFFER_SIZE     1024
 
+#ifndef PATH_MAX
+ #define PATH_MAX 4096
+#endif
+
 struct kanatest_prefs
 {
     gint        window_x;
